schemas/expconf/v0/environment-image.json

```diff
@@ -3,7 +3,7 @@
     "$id": "http://determined.ai/schemas/expconf/v0/environment-image.json",
     "title": "EnvironmentImage",
     "union": {
-        "defaultMessage": "is neither a string nor a map of cpu/gpu to strings",
+        "defaultMessage": "is neither a string nor a map of cpu, cuda/gpu, or rocm to strings",
```

Comment:
Can we remove the deprecated name `gpu` from the error message?  I want people who see this message to be clearly guided by it toward the modern usage.